guidelines/index.html

```diff
@@ -993,9 +993,9 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
  
  <p>例外: テキスト表記においてこれらのテキストスタイルプロパティの一つ以上を使用しない自然言語及び文字体系では、その言語と文字体系の組み合わせに存在するプロパティだけを用いて、この達成基準に適合することができる。</p>
 
- <div class="note" role="note" id="issue-container-generatedID-10"><div role="heading" class="note-title marker" id="h-note-10" aria-level="5"><span>注記 1</span></div><p class="">Content is not required to use these text spacing values. The requirement is to ensure that when a user overrides the authored text spacing, content or functionality is not lost.</p></div>
+ <div class="note" role="note" id="issue-container-generatedID-10"><div role="heading" class="note-title marker" id="h-note-10" aria-level="5"><span>注記 1</span></div><p class="">コンテンツが、これらのテキストの間隔の値を用いる必要はない。要求事項は、制作されたテキストの間隔を利用者が上書きした場合に、コンテンツ又は機能が失われないことである。</p></div>
```

Comment:
the authored text spacing 「制作されたテキストの間隔」は日本語の表現として少し引っ掛かります。少し補って「制作者が指定したテキストの間隔」くらいにしてはどうでしょうか。

```suggestion
 <div class="note" role="note" id="issue-container-generatedID-10"><div role="heading" class="note-title marker" id="h-note-10" aria-level="5"><span>注記 1</span></div><p class="">コンテンツが、これらのテキストの間隔の値を用いる必要はない。要求事項は、制作者が指定したテキストの間隔を利用者が上書きした場合に、コンテンツ又は機能が失われないことである。</p></div>
```